clients/src/main/java/org/apache/kafka/clients/producer/KafkaProducer.java

```diff
@@ -156,7 +155,7 @@
  * to multiple partitions (and topics!) atomically.
  * </p>
  * <p>
- * To enable idempotence, the <code>enable.idempotence</code> configuration must be set to true. If set, the
+ * In Kafka 3.0, the <code>enable.idempotence</code> configuration is default to true. When enabling idempotence,
```

Comment:
`In Kafka 3.0,` -> `From Kafka 3.0,`
`is default to true` -> `defaults to true`